packages/wallets/src/features/cashier/modules/WithdrawalCrypto/components/WithdrawalCryptoForm/WithdrawalCryptoForm.tsx

```diff
@@ -63,8 +27,7 @@ const WithdrawalCryptoForm: React.FC<TWithdrawalCryptoFormProps> = ({
             onSubmit={values =>
                 requestCryptoWithdrawal({
                     address: values.cryptoAddress,
-                    amount: parseFloat(parseFloat(values.cryptoAmount).toFixed(FRACTIONAL_DIGITS_CRYPTO)),
-                    verification_code: verificationCode,
+                    amount: parseFloat(parseFloat(values.cryptoAmount).toFixed(fractionalDigits.crypto)),
```

Comment:
I know it was already in that PR before, but thought it might be worth pointing out for future - the most bulletproof implementation of floating point precision handling is loadash.round (as internally, it does use scientific notation, so just like the the binary representation of floating point in memory)

https://lodash.com/docs/#round